queue_services/entity-emailer/src/entity_emailer/config.py

```diff
@@ -74,6 +74,7 @@ class _Config():  # pylint: disable=too-few-public-methods
     PAY_API_URL = os.getenv('PAY_API_URL', None)
     AUTH_URL = os.getenv('AUTH_URL', None)
     ACCOUNT_SVC_AUTH_URL = os.getenv('ACCOUNT_SVC_AUTH_URL', None)
+    AUTH_SVC_URL = os.getenv('AUTH_URL', 'https://')
```

Comment:
we have it in line 75

Reply:
Yes, we have it. But, the emailer import the legal-api.service that use as AUTH_SVC_URL not AUTH_URL
https://github.com/bcgov/lear/blob/881ad4d19940bd0f7e119d4fe9e19a55d08a2582/legal-api/src/legal_api/services/bootstrap.py#L282

Reply:
removed